flink-container/src/main/java/org/apache/flink/container/entrypoint/StandaloneJobClusterEntryPoint.java

```diff
@@ -97,14 +102,26 @@ public static void main(String[] args) {
 
 		StandaloneJobClusterEntryPoint entrypoint = new StandaloneJobClusterEntryPoint(
 			configuration,
-			clusterConfiguration.getJobId(),
+			createJobIdForCluster(Optional.ofNullable(clusterConfiguration.getJobId()), configuration),
```

Comment:
Personally, I'd change it to:
```
Optional.ofNullable(clusterConfiguration.getJobId())
    .orElseGet(() -> createJobIdForCluster(configuration))
```

Reply:
I think this is a good improvement because it simplifies `createJobIdForCluster`